documentation/diff-viewer/end-user-guide.md

```diff
@@ -0,0 +1,83 @@
+#Diff Viewer End User Guide
```

Comment:
@danielsuter Just a minor remark: "End User" is a term that I do not like. Wouldnt just "User" and therefor "User Guide" be enough? "End User" sounds strange to me ... in german it is even worse, also reminds me of some dark german history.